contrib/storage-opentsdb/src/test/java/org/apache/drill/store/openTSDB/TestOpenTSDBPlugin.java

```diff
@@ -51,17 +54,17 @@
 
 public class TestOpenTSDBPlugin extends PlanTestBase {
 
-  protected static OpenTSDBStoragePlugin storagePlugin;
-  protected static OpenTSDBStoragePluginConfig storagePluginConfig;
+  private static int portNum = 10_000;
```

Comment:
Why do you set value right away? It looks you will always re-write in `setup`.

Reply:
Thanks, removed.